src/p2pcf.js

```diff
@@ -74,11 +70,8 @@ const webkit = !!ua.match(/WebKit/i)
 const iOSSafari = !!(iOS && webkit && !ua.match(/CriOS/i))
 const isFirefox = !!(navigator?.userAgent.toLowerCase().indexOf('firefox') > -1)
 
-const { decode: base64ToArrayBuffer } = require('base64-arraybuffer')
-const arrayBufferToHex = require('array-buffer-to-hex')
-
-const hexToBase64 = hex => arrayBufferToBase64(hexToBytes(hex))
-const base64ToHex = b64 => arrayBufferToHex(base64ToArrayBuffer(b64))
+const hexToBase64 = hex => btoa(hex)
```

Comment:
Unfortunately I don't think this isn't going to work - the DTLS fingerprints in SDP are hex encoded not base64 encoded. The dependencies for these bits are pretty tiny, so opted for pulling them in vs inlining them. If it looks like they are pulling in a lot of bloat then lmk. (I might have missed sometihng.)

Reply:
yup, I was thinking about a different hex... i was hoping to strip some dependencies, but it probably wont happen like that, at most i can replace those deps with something marginally faster, but it's probably not worth the effort

Reply:
👍